cardano-cli/test/Test/Cli/Pioneers/Exercise2.hs

```diff
@@ -35,7 +35,10 @@ prop_createTransaction = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
   -- Create transaction body
   void $ execCardanoCLI
     [ "transaction", "build-raw"
-    , "--tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "--script-file", "test/data/golden/shelley/multisig/scripts/all"
+    , "tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "--script-file", "test/data/golden/shelley/multisig/scripts/all"
```

Comment:
We've converted the `--tx-in` flag into the `tx-in` sub-command. We need to look again at how we do this with ordinary flags.